src/pybasic/tools/dct2d_tools.py

```diff
@@ -0,0 +1,29 @@
+"""2D discrete consine transform tools."""
+
+import numpy as np
+
+
+def dct2d(im_stack: np.ndarray) -> np.ndarray:
+    """Calculates 2D discrete cosine transform.
+
+    Args:
+        im_stack: input image stack
+
+    Returns:
+        transformed image stack
+    """
+    ...
+    return
+
+
+def idct2d(im_stack: np.ndarray) -> np.ndarray:
+    """Calculates 2D inverse discrete cosine transform.
+
+    Args:
+        im_stack: input image stack
+
+    Returns:
+        transformed image stack
+    """
+    ...
+    return

```

Comment:
We should be specific about what class of DCT we are using in the documentation.